ReSwift/CoreTypes/Store.swift

```diff
@@ -204,4 +212,17 @@ extension Store where State: Equatable {
             }
             _ = subscribe(subscriber, transform: { $0.skipRepeats() })
     }
+
+    open func subscribe<SelectedState: Equatable, S: StoreSubscriber>(
```

Comment:
This looks pretty good. But wouldn't it be possible to implement this method as part of the extension (`extension Store where State: Equatable`)? That might be more in line with the existing implementation.

(I'm traveling right now, so can't come up with what the actual implementation would look like 😉 )

Reply:
See comment below; this can be disregarded.